openapi/resource-server.yaml

```diff
@@ -1012,6 +1020,22 @@ components:
         - sentAmount
         - createdAt
         - updatedAt
+    outgoing-payment-with-spent-amounts:
+      title: Outgoing Payment With Grant Spent Amounts
+      description: 'An **outgoing payment** resource represents a payment that will be, is currently being, or has previously been, sent from the wallet address.'
+      allOf:
+        - $ref: '#/components/schemas/outgoing-payment'
+        - type: object
+          properties:
+            grantSpentDebitAmount:
+              $ref: './schemas.yaml#/components/schemas/amount'
+              description: The amount already deducted from the sender's account against the outgoing payment.
+            grantSpentReceiveAmount:
+              $ref: './schemas.yaml#/components/schemas/amount'
+              description: The total amount already received against the outgoing payment.
+          required:
+            - grantSpentDebitAmount
+            - grantSpentReceiveAmount
```

Comment:
Seeing an error in CI: 

> Error:  259:25  error  oas3-valid-media-example  Property "grantSpentDebitAmount" is not expected to be here  paths./outgoing-payments.post.responses[201].content.application/json.examples['New Fixed Send Outgoing Payment for $25'].value

If I comment them out, then it actually tells me it expects these properties. It's like the validator expects the `POST /outgoing-payments` example to fulfill each of the `allOf`. That seems reasonable except it doesnt appear to work that way for our `incoming-payment-with-methods` component.

After investigating further I found that this is because `incoming-payment` allows `additionalProperties` (it's not set and defaults to true).

Reply:
Ended up in-lining all the outgoing payment properties https://github.com/interledger/open-payments/pull/474/commits/f74588257d004f9f682a62f2e75941e6fe9f3c2e